metecho/api/models.py

```diff
@@ -393,6 +393,8 @@ class Project(
     status = models.CharField(
         max_length=20, choices=PROJECT_STATUSES, default=PROJECT_STATUSES.Planned
     )
+    # List of strings:
+    available_task_org_config_names = JSONField(default=list, blank=True)
```

Comment:
@wlonk Based on David's example (https://trello.com/c/Eta19Ib5/188-i-want-to-create-a-scratch-org-with-a-different-flow-or-org-type#comment-5ec2dac8e800a381d859b074), this isn't just a list of strings. We want to support a label `[key].label` and a description `[key].description`, with fallbacks of `label: [key]` and `description: ''`. So likely a list of objects, each with `label` and `description` keys?

Reply:
Or actually, we also need the value/key exposed as well as the optional label and description.